core/route/allrouting.js

```diff
@@ -87,6 +87,18 @@ class Routing {
     }
 
     /**
+     * Match Method
+     * @param {String[]} methods - Array of methods to use for a route
+     * @param {String} path - Path for routing
+     * @param {String|Function} controllerAction
+     */
+    match(methods, path, controllerAction) {
+        methods.forEach((method) => {
+            this[method.toLowerCase()](path, controllerAction);
```

Comment:
I was planning on making error handling a full task for the future. Good errors handling is a big help.